src/app/models/ability.rb

```diff
@@ -2,6 +2,6 @@ class Ability
   include CanCan::Ability
 
   def initialize(user)
-    can :index, [Event, Participant, Timeslot]
+    can :index, [Event, Participant, Timeslot, MarkdownContent]
```

Comment:
I've been in Pundit-land lately... I can't remember how this works... Do we also need to give Admins the ability to access/edit/update/etc MarkdownContent? How is that managed?

Reply:
For this project it doesn't look to be enforced. All of the admin controllers are using `load_resource` instead of `load_and_authorize_resource` which isn't great, but all of the admin controllers require the caller be an admin.